static/pages/home/home.html

```diff
@@ -83,7 +83,8 @@ <h1>
       <div class="macbook">
         <div class="screen">
           <div class="viewport">
-            <video controls class="home-page-interactions-video">
+            <video controls controlsList="nodownload"
+                  class="home-page-interactions-video">
```

Comment:
nit: indent by 1 more to match "controls" in previous line

Reply:
Thanks for spotting this :)